openstack/designate/values.yaml

```diff
@@ -187,3 +176,22 @@ logging:
         sqlalchemy:
             handlers: stdout, sentry
             level: WARNING
+
+audit:
+  enabled: true
+  # do by default not attach the request payloads of create/update calls to the event
+  record_payloads: false
+  metrics_enabled: true
+  # how many messages to buffer before dumping to log (when rabbit is down or too slow)
+  mem_queue_size: 1000
+
+rabbitmq_notifications:
+  name: designate
+  # overwrite auth in region secrets
```

Comment:
the default user-names are defined by the rabbitmq chart's value file already. Let's keep them unchanged to avoid confusion and remove this section here.

Reply:
done